_opensearch/install/rpm.md

```diff
@@ -5,154 +5,472 @@ parent: Install OpenSearch
+   ```bash
+   # Change to the correct directory
+   cd /usr/share/opensearch/plugins/opensearch-security/tools
+   ```
+1. Invoke the script. See [Apply changes using securityadmin.sh]({{site.url}}{{site.baseurl}}/security-plugin/configuration/security-admin/) for definitions of the arguments you must pass.
    ```bash
-   sudo systemctl stop opensearch.service
-   sudo systemctl stop opensearch-dashboards.service
+   # You can omit the environment variable if you declared this in your $PATH.
+   OPENSEARCH_JAVA_HOME=/usr/share/opensearch/jdk ./securityadmin.sh -cd /etc/opensearch/opensearch-security/ -cacert /etc/opensearch/root-ca.pem -cert /etc/opensearch/admin.pem -key /etc/opensearch/admin-key.pem -icl -nhnv
    ```
 
+### Verify that the service is running
 
-## *(Optional)* Set up Performance Analyzer
+OpenSearch is now running on your host with custom TLS certificates and a secure user for basic authentication. You can verify external connectivity by sending an API request to your OpenSearch node from another host.
 
-When enabled, the Performance Analyzer plugin collects data related to the performance of your OpenSearch instance. To start the Performance Analyzer plugin, enter:
+During the previous test you directed requests to `localhost`. Now that TLS certificates have been applied and the new certificates reference your host's actual DNS record, requests to `localhost` will fail the CN check and the certificate will be considered invalid. Instead, requests should be sent to the address you specified while generating the certificate.
 
-```bash
-sudo systemctl start opensearch-performance-analyzer.service
-```
-
-To stop the Performance Analyzer, enter:
+You should add trust for the root certificate to your client before sending requests. If you do not add trust, then you must use the `-k` option so that cURL ignores CN and root certificate validation.
+{:.tip}
 
 ```bash
-sudo systemctl stop opensearch-performance-analyzer.service
+$ curl https://your.host.address:9200 -u admin:yournewpassword -k
+{
+  "name" : "hostname-here",
+  "cluster_name" : "opensearch",
+  "cluster_uuid" : "efC0ANNMQlGQ5TbhNflVPg",
+  "version" : {
+    "distribution" : "opensearch",
+    "number" : "2.1.0",
+    "build_type" : "tar",
+    "build_hash" : "388c80ad94529b1d9aad0a735c4740dce2932a32",
+    "build_date" : "2022-06-30T21:31:04.823801692Z",
+    "build_snapshot" : false,
+    "lucene_version" : "9.2.0",
+    "minimum_wire_compatibility_version" : "7.10.0",
+    "minimum_index_compatibility_version" : "7.0.0"
+  },
+  "tagline" : "The OpenSearch Project: https://opensearch.org/"
+}
 ```
 
-## Upgrade RPM
+## Upgrade to a newer version
 
-You can upgrade your RPM OpenSearch instance both manually and through YUM. 
+OpenSearch instances installed using RPM or YUM can be easily upgraded to a newer version. We recommend updating with YUM, but you can also upgrade using YUM.
```

Comment:
Check the second sentence for accuracy. Is "YUM" meant to be repeated? It looks like we're perhaps differentiating between updating and upgrading, but the sentence should probably be clearer.